src/DynamoCoreWpf/Utilities/CrashUtilities.cs

```diff
@@ -70,11 +76,13 @@ internal static string BuildMarkdownContent(string dynamoVersion, string stackTr
                 "(Fill in here)" + Environment.NewLine + Environment.NewLine +
 
                 "## What packages or external references (if any) were used?" + Environment.NewLine +
-                "(Fill in here)" + Environment.NewLine + Environment.NewLine + Environment.NewLine +
+                "(Fill in here)" + Environment.NewLine + Environment.NewLine +
+
+                markdownPackages + Environment.NewLine + Environment.NewLine +
 
                 "## Stack Trace" + Environment.NewLine +
                 "```" + Environment.NewLine +
-                "(From the Dynamo crash window select 'Details' -> 'Copy' and paste here)" + Environment.NewLine +
+                stackTrace + Environment.NewLine +
```

Comment:
@QilongTang I notice the stack trace information was already available to be included in the issue template but was never used so I modified it to be automatically added instead of asking the user to copy and paste the contents of the crash prompt. If there was any specific reason this wasn't done already I can revert the change, just let me know.

Reply:
hi @Astul-Betizagasti I think certain case that the crash detail is too long to be sent automatically. That was what I remember. @alfarok should be able to confirm why we did not include it then

Reply:
ah I see, well let me know if it should be reverted, and in case it does, should I remove the stack trace being sent to the method all together? It could make for a cleaner code if we are not going to use it

Reply:
I can't think of why we would not need the stack trace. I think in almost all cases we ask reporters to paste the stack trace in the issue where it's missing so if it can be done automatically there's nothing better than that.

Reply:
Yes I agree, I didn't meant removing it from the crash prompt, I meant not sending it through the event when clicking on the button to generate the GitHub issue as a parameter
![image](https://user-images.githubusercontent.com/66637541/107801644-1bb2c500-6d3f-11eb-98d3-9e1ce247c6ef.png)
At the moment that's how the method looks and the `this.CrashDetailsContent.Text` parameter has the stack trace information that will not be used, while it could have some other info like the loaded packages for example

Reply:
Ah ok, I remember there was an issue with this. @QilongTang is right.

Reply:
Thanks, I would remove the stacktrace here. Crashes callstacks or VM crashes callstacks are covered by analytics as a backup, I would not include here just to make sure user can always log an issue when they want. If anything interrupts that, it may discourage them from doing that later.